### ISSUE TITLE
Unable to get TIMEX3 labels.

### ISSUE THREAD

**Issue by edemattos** [paulrinckens/timexy#2]
I've tried installing `timexy` on both macOS 12 Monterrey and Ubuntu 18 LTS, and running the sample code provided on the README but am not able to get the TIMEX3 descriptors.

```python
# copied from README

import spacy
from timexy import Timexy

nlp = spacy.load("en_core_web_sm")

# Optionally add config if varying from default values
config = {
    "kb_id_type": "timex3",  # possible values: 'timex3'(default), 'timestamp'
    "label": "timexy",  # default: 'time'
    "overwrite": False  # default: False
}
nlp.add_pipe("timexy", config=config)

doc = nlp("Today is the 10.10.2010. I was in Paris for six years.")
for e in doc.ents:
    print(f"{e.text}\t{e.label_}\t{e.kb_id_}")    
```

```bash
$ conda create --name timexy python=3.8
$ conda activate timexy
$ pip install spacy timexy
$ python -m spacy download en_core_web_sm
$ python timexy.py
Today	DATE	
10.10.2010	DATE	
Paris	GPE	
six years	DATE
```

**Comment by paulrinckens** [comment]
Hi @edemattos!

The entities that you obtain are of type DATE and not created by timexy but by the ner component of the en_core_web_sm model. The example in the readme is missing to set the overwrite flag to `True`. Thanks for reporting this @edemattos!

To get the expected timexy entites instead, either disable the ner component in the en_core_web_sm model 
```python
nlp = spacy.load("en_core_web_sm", disable=["ner"])
```

or set the overwrite flag on the timexy component
```python
config = {
    "kb_id_type": "timex3",  # possible values: 'timex3'(default), 'timestamp'
    "label": "timexy",  # default: 'time'
    "overwrite": True  # default: False
}
nlp.add_pipe("timexy", config=config)
```

**Comment by edemattos** [comment]
That does the trick, thanks!

**Comment by paulrinckens** [comment]
Fixed by #3
